nfsstat/datadog_checks/nfsstat/nfsstat.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, name, init_config, agentConfig, instances):
         AgentCheck.__init__(self, name, init_config, agentConfig, instances)
         # if they set the path, use that
         if init_config.get('nfsiostat_path'):
-            self.nfs_cmd = [init_config.get('nfsiostat_path'), '1', '2']
+            self.nfs_cmd = init_config.get('nfsiostat_path').split() + ['1', '2']
```

Comment:
(Nit) Since we already check if `nfsiostat_path` is set in the if statement…

```suggestion
            self.nfs_cmd = init_config['nfsiostat_path'].split() + ['1', '2']
```